src/main/java/example/Example.java

```diff
@@ -0,0 +1,11 @@
+package example;
+
+import com.google.common.base.Joiner;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](http://54.201.239.26:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarQube](http://54.201.239.26:9000/project/issues?id=gradle-build-scan-quickstart&pullRequest=144&issues=AWfNAg3rv9TGX2wD8SVd&open=AWfNAg3rv9TGX2wD8SVd)

[ISSUE_KEY]: AWfNAg3rv9TGX2wD8SVd